src/Demos/12-norvig's-spelling-corrector.fsx

```diff
@@ -58,7 +58,7 @@ let files = downloadJob.AwaitResult()
 
 let fileSizesJob = 
     files
-    |> Distributed.map CloudFile.GetSize
+    |> DivideAndConquer.map CloudFile.GetSize
```

Comment:
Specifically we should add a version number [here](https://github.com/elastacloud/mbrace-on-brisk-starter/blob/master/src/Demos/paket.references) and have @isaacabraham update it when Brisk updates its default version